### PR TITLE
65 dicover heatmap endpoint

### PR DESCRIPTION
**Pull request by andriwild**

### CODE REVIEW

File: services/api/routers/discover.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    limit_per_day: int = 3) -> list[BirdSpeciesCount]:
+
+    time_from_condition = "AND time >= :time_from" if time_from else ""
+    time_to_condition = "AND time <= :time_to" if time_to else ""
+
+    query = text(f"""
+    WITH ranked_results AS (
+        SELECT
+            br.species,
+            time_bucket(:bucket_width, time) AS bucket,
```
Comment (cspindler) [inline services/api/routers/discover.py:63]: You're using the timescaledb function `time_bucket(interval, timestamptz)` here. When I try to run this query, Postgres complains that this function doesn't exist. How did you get this to work?

(It has been used before in other endpoints, and it is supposed to work. It just seems important to clarify why and how it works)

Comment (cspindler) [reply services/api/routers/discover.py:63]: Never mind, it seems to be a scope issue of a specific SQL client.

File: services/api/routers/discover.py
Comment (cspindler) [inline services/api/routers/discover.py:1]: I changed the spelling `convidence` to `confidence`. Does this break the discover app?

**Review by cspindler — CHANGES_REQUESTED** [review]
Confirm the spelling change of `convidence` doesn't break the discover app.

File: services/api/routers/discover.py
Comment (andriwild) [reply services/api/routers/discover.py:1]: Thanks, it was just a typo.

**Pull request merged**